subsys/bluetooth/host/direction.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_SUBSYS_BLUETOOTH_HOST_DF_H_
+#define ZEPHYR_SUBSYS_BLUETOOTH_HOST_DF_H_
+
+#include <sys/util.h>
+
+enum df_opt_swich_sample_rates {
+	/* Support Angle of Departure 1us switching TX */
+	DF_AOD_1US_SWITCH_TX_SUPP = BIT(0),
+	/* Support Angle of departure 1us sampling RX */
+	DF_AOD_1US_SAMPL_RX_SUPP = BIT(1),
+	/* Support Angle of Arrival 1us switching and sampling RX */
+	DF_AOA_1US_SWITCH_SAMPL_RX_SUPP  = BIT(2),
+};
+
+/* @brief Antenna information for LE Direction Finding */
+struct bt_le_df_ant_info {
```

Comment:
Should hhis definition be in a public header?

Reply:
Well, there is no public header because there is no requirement to provide public interface. At least I haven't seen any API proposal for that. Because of that I decided to provide some internal command wrappers to be re-used in future.